### ISSUE TITLE
getProducts() it returns empty. iOS

### ISSUE THREAD

**Issue by burton22ful** [AlexDisler/cordova-plugin-inapppurchase#223]
When calling getProducts() it returns empty.  I have the correct product ID and I think i have set everything correct in itunes connect.

My question is in regards to this -->You did not complete all the financial requirements. See Agreements, Tax, and Banking Information for more information.

I have requested and submitted the contract and it says pending Tax and Bank Info.  

Do i need to supply these details first before it will work?

This means i will need to provide an ABN(Australian Business Number) and GST Number before i can start testing in app purchases?  Is this why its not working?

I have this already working on android so i know the code works. The problem is with IOS.

Please include:

### Your code / steps to reproduce
 inAppPurchase
  .getProducts(itemsArr)
  .then(function (products) {
   	 	var strItem = JSON.stringify(products)
	 	alert ('strItemz ' + strItem)
  }

### Type of product you are working with consumable/non-consumable/subscription
consumable

### Version of cordova
8

### Version of iOS/Android
IOS

**Comment by burton22ful** [comment]
console output is

RMStore: products request received response
RMStore: invalid product with id xx.xxx.xxxxxxx

**Comment by shochberg** [comment]
Yes, although Apple doesn't tell you this, you have to complete the contract, bank and tax info before you can access your products, even in sandbox.  However, you do not need to submit your binary for approval if you stay in sandbox.

I just went through this same frustration and got completely incorrect info from Apple, but once I completed all 3 pieces of the setup in the contracts section, it worked great.

**Comment by FiveHundredMiles** [comment]
Please check #148 .
